modules/hub/pages/autoconnect-mode.adoc

```diff
@@ -0,0 +1,51 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+In this example, autoconnect mode is used, this is a convenient method where it is assumed that each server has an admin user with the same credentials as one of Hub's user.
```

Comment:
I would reword it as something like:

The `autoconnect` mode is a convenient method where the same credentials used to login into the Hub API are used to login into the APIs of the servers the user wants to work with. It is assumed the credentials are valid in each server and correspond to an admin user.